ddl/BUILD.bazel

```diff
@@ -0,0 +1,229 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "ddl",
+    srcs = [
+        "backfilling.go",
```

Comment:
Shall we manually update this list everytime we add, delete or rename files?

Reply:
No, This job is automatic.

Reply:
Thanks for your information. You may create an issue on https://github.com/pingcap/tidb-dev-guide later and add some words about working with bazel.

Reply:
[a good examle](https://wiki.crdb.io/wiki/spaces/CRDB/pages/2221703221/Developing+with+Bazel)